### PR TITLE
Remove redundant word

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for #10971

As requested by @ot2sen and others this PR removes the redundant word successfully from success messages. It's just not needed at all. You can't have a success message that says you "unsuccessfully" did something so there is no need to use the word.

Note for readability where a sentence began with the word successfully I left it there.

This can be merged after 3.7.0 if requested - this is upto @bakual as commented at #10971

### CODE REVIEW

**Comment by Bakual** [comment]
Thanks.
I have added the 3.8.0 milestone for now.

File: administrator/language/en-GB/en-GB.com_contenthistory.ini [hunk 1]
```diff
@@ -33,8 +33,8 @@ COM_CONTENTHISTORY_COMPARE_VALUE2="Saved on %s %s"
 COM_CONTENTHISTORY_ERROR_KEEP_NOT_PERMITTED="You are not permitted to change the keep forever status."
 COM_CONTENTHISTORY_KEEP_VERSION="Keep Forever"
 COM_CONTENTHISTORY_MODAL_TITLE="Item Version History"
-COM_CONTENTHISTORY_N_ITEMS_DELETED_1="%s history version successfully deleted."
-COM_CONTENTHISTORY_N_ITEMS_DELETED="%s history versions successfully deleted."
+COM_CONTENTHISTORY_N_ITEMS_DELETED_1="%s history version deleted."
+COM_CONTENTHISTORY_N_ITEMS_DELETED="%s history versions deleted."
 COM_CONTENTHISTORY_N_ITEMS_KEEP_TOGGLE_1="Successfully changed the keep forever value for %s history version."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_contenthistory.ini]: Remove `Successfully`.

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_contenthistory.ini]: No. It makes it correct English to begin with the word. It is not redundant here.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 97b3e47c181946962f1e2870f2ff9e3a551ccd65<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14607">issues.joomla.org/tracker/joomla-cms/14607</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 97b3e47c181946962f1e2870f2ff9e3a551ccd65<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14607">issues.joomla.org/tracker/joomla-cms/14607</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by zero-24** [comment]
@brianteeman can you fix the conflicts?

**Comment by brianteeman** [comment]
It has a milestone of 3.8 so I really only want to fix conflicts once. If someone can say when it will be merged in to 3.8 I will fix the conflicts (or try to) in time for that

**Comment by zero-24** [comment]
Cant you just change the base branch? And that we can merge it into 3.8-dev?

**Comment by brianteeman** [comment]
I dont see how to change the base branch?

**Comment by brianteeman** [comment]
Conflicts resolved

**Comment by zero-24** [comment]
you can do this by using the edit button. but this require the staging and 3.8-dev branch to be in sync (i have just reyed it). I think I can take a look into that after 3.7.0 stable is out.

**Comment by brianteeman** [comment]
base branch changed

**Comment by zero-24** [comment]
Yes but now your PR contains 40 file changes (most are not realted to your PR) https://github.com/joomla/joomla-cms/pull/14607/files

**Comment by brianteeman** [comment]
ah - oops - changed back to staging now.

**Comment by zero-24** [comment]
Thanks. I'm going to check that later. IIRC we just need to sync up staging with 3.8-dev and we do that again.

**Comment by Bakual** [comment]
@zero-24 I will handle this one after 3.7.0 is released. I need to sync it correct with Crowdin anyway to avoid 321 changed strings there :smile:

**Comment by zero-24** [comment]
Handed over than. Thanks!

**Comment by rdeutz** [comment]
what is with the "COM_CONTACT_N_ITEMS_(UN)FEATURED" Tags?

**Comment by brianteeman** [comment]
conflicts resolved - it was a pr you just merged

**Comment by brianteeman** [comment]
thanks for merging

@bakual pinging you to do whatever it is you wanted to do on crowdin

**Comment by Bakual** [comment]
Hope it worked :)

**Comment by 810** [comment]
Can we revert this PR, because many tests are broken because this change. We should change this later, not on minor versions.

**Comment by brianteeman** [comment]
what tests?

**Comment by 810** [comment]
Codeception tests.

See: https://github.com/joomla-projects/joomla-browser/issues/133

**Comment by brianteeman** [comment]
there are 17 changes in total to make and it will take you about 5 minutes

**Comment by Bakual** [comment]
> We should change this later, not on minor versions.

This change is absolutely fine for patch versions (it's not even a minor version).
If the codeception tests fail, they need to be fixed.

On a sidenote: As long as those tests don't run automatically on Pull Requests, we cant make decisions based on those tests anyway.

**Comment by 810** [comment]
But our componens/modules/plugins/templates are running on the pr's, so we are having failed tests now.
We need to improve the joomla browser and our own tests on each rep.

I will improve it, but it was better todo on J3.8, now we have b/c, some 3rth parties will have this issue.

**Comment by mbabker** [comment]
Why do we have a test suite that is supposedly testing the core package not integrated into this repository in any way (either the tests living in our tests directory or hooked into the CI process)?  Regardless of when this PR was accepted and where the tests live, it all would have to be updated anyway.  I don't see a strong argument for reverting language improvements because some tooling requires updates to work with those changes.

**Comment by brianteeman** [comment]
what is the point in having a test suite if its not integrated. seems to me that it means that anyoone working on the test suite is wasting their time

**Comment by Bakual** [comment]
> now we have b/c, some 3rth parties will have this issue.

It's no backward compatiblity break if language strings get improved. Nothing will break except some tests which expects a specific string. But that's a "false positive" as in production nothing wil break at all.

**Comment by mbabker** [comment]
And in terms of updating tests it's no worse than having to update a test validating HTML output when the method being tested gets updated.

**Comment by brianteeman** [comment]
The tests could have been updated by now. ;)

I would have done it myself if I wasn't on a train 🚂 crossing a border

**Pull request merged**